### DISCUSSION TITLE
Validation at submit button is overlapping

### DISCUSSION THREAD

**Discussion by Jeeva-Mindwave** [apostrophecms/apostrophe#2871] [General]
FYR I have attached the screenshot:

![Screenshot from 2021-04-01 15-47-17](https://user-images.githubusercontent.com/80744312/113280550-0fd88d80-9302-11eb-8a16-47db0b13bc9c.png)

Please resolve this for better quality

**Comment by abea** [comment]
Thank you. We've passed this along to the right person internally.
